### ISSUE TITLE
Swatch isn't selected when minicolors inits and input has a swatch color value

### ISSUE THREAD

**Issue by pjfsilva** [claviska/jquery-minicolors#256]
Steps to reproduce:

On an input field with a value of a color of a swatch, init the minicolors plugin. If you open minicolors you can see the swatch color isn't selected (if you type another value it is then selected)

This happens because on init updateFromInput is called and that doesn't do the logic of selecting the swatch instead it calls doChange but only when minicolors is initialized:

```
// Fire change event, but only if minicolors is fully initialized
    if(input.data('minicolors-initialized')) {
      doChange(input, value, opacity);
    }
```

I'm not sure why doChange isn't called always, if it's just for performance issue or if it's to fix some bugs but I see 2 possible solutions: add swatch selection logic to init or updateFromInput or always call doChange even if minicolors is initialiazing.

**Comment by claviska** [comment]
Good catch. `doChange()` will trigger a change event, so it's probably best to set this on init.

Care to submit a PR? 🙏
